Src/EmbeddedScripts.Python.PythonNet/EmbeddedScripts.Python.PythonNet.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>netstandard2.0</TargetFramework>
+        <RootNamespace>EmbeddedScripts.Python.PythonNet</RootNamespace>
+        <LangVersion>latest</LangVersion>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <PackageReference Include="pythonnet_netstandard_py39_win" Version="2.5.2" />
```

Comment:
Probably will need a linux-x64 version of python runtime to get green tests

Reply:
Situation is more complicated. I have just copied python's runtime into bin/Debug folder on Windows